tests/css-borders-4.js

```diff
@@ -149,49 +186,438 @@ export default {
 			},
 			tests: [
 				'round',
-				'angle',
-				'round angle',
-				'round angle round',
-				'round angle round angle',
+				'scoop',
+				'bevel',
+				'notch',
+				'straight',
```

Comment:
Rename `straight` to `square`

CSSWG resolution: https://github.com/w3c/csswg-drafts/issues/11607#issuecomment-2769753118

Reply:
Thanks for the hint! Done now.